src/modules/new-request-form/translations/en-us.yml

```
@@ -17,6 +25,7 @@ parts:
       title: "Title of the footer of the Answer Bot Modal"
       screenshot: "https://drive.google.com/file/d/14qsEyUiVTy9MRYBGkgVWLmlduhoscYBw/view?usp=drive_link"
       value: "Does this article answer your question?"
+      obsolete: "2024-08-22"
```

```suggestion
      obsolete: "2024-09-22"
```
The date should be at least 2 months in the future :)

---

Thanks, changed